pkgs/development/web/mailcatcher/default.nix

```diff
@@ -1,29 +1,11 @@
-{ stdenv, bundlerEnv, ruby, makeWrapper }:
+{ lib, bundlerApp }:
 
-stdenv.mkDerivation rec {
-  name = "mailcatcher-${version}";
+bundlerApp {
```

Comment:
I was not aware of this feature. Is this documented in our nixpkgs manual?
I know that we have many packages are not make use of the `exes` attribute.

Reply:
it is: https://nixos.org/nixpkgs/manual/#sec-language-ruby

Reply:
I hope the function around bundler can be refactored a bit. I found them quite hard to follow.